src/System.Text.Encodings.Web/ref/System.Text.Encodings.Web.cs

```diff
@@ -0,0 +1,238 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+#pragma warning disable CS3011 // Only CLS-compliant members can be abstract
+
+namespace System.Text.Encodings.Web
+{
+    public abstract partial class HtmlEncoder : TextEncoder
```

Comment:
Can the ref be auto-generated instead? If it is already, should `TextEncoder` be fully qualified?